src/Sentry.AspNetCore/SentryTracingMiddleware.cs

```diff
@@ -50,6 +50,30 @@ internal class SentryTracingMiddleware
             }
         }
 
+        private BaggageHeader? TryGetBaggageHeader(HttpContext context)
```

Comment:
Is there any difference between this one and the one in `HttpContextExtensions`? If not, maybe we can extract to an extension method and reuse the `BaggageHeader` in different places?

Reply:
This one is for ASP.NET Core, and the other one is for ASP.NET (framework).

The only real difference between them is in how we retrieve the header from the request, which differs between the two flavors of ASP.NET.

However, we don't currently share code between them in a way that would be useful here.